backend/app/views/spree/admin/customer_returns/_return_item_selection.html.erb

```diff
@@ -38,7 +38,9 @@
           <%= return_item.display_amount %>
         </td>
         <td class="align-center">
-          <%= return_item.inventory_unit.state %>
+          <span class="pill pill-<%= return_item.inventory_unit.state %>">
+            <%= return_item.inventory_unit.state.titleize %>
```

Comment:
Same comment as above. I suggest to make a separate commit (not even a PR) first that changes the presentation via `I18n`.